src/lib.rs

```diff
@@ -148,25 +155,29 @@ async fn get_migrated(db: &mut PgConnection) -> Result<Vec<String>> {
     }
 }
 
+const DEFAULT_MIGRATION_TABLE: &str = "sqlx_pg_migrate";
+
 /// Runs the migrations contained in the directory. See module documentation for
 /// more information.
-pub async fn migrate(url: &str, dir: &Dir<'_>) -> Result<()> {
+pub async fn migrate(url: &str, dir: &Dir<'_>, table: Option<&str>) -> Result<()> {
```

Comment:
Instead of adding options one by one, I think a better approach will be to have this simpler migrate method that takes the two required parameters, and another method that takes an `Options` struct, which for now will just contain the migration table name. The simpler migrate function will call the other with the default value.